src/main/kotlin/domain/Coordinate.kt

```diff
@@ -4,8 +4,25 @@ data class Coordinate(
     val row: Int,
     val col: Int,
 ) {
+    operator fun plus(other: Coordinate) = Coordinate(row + other.row, col + other.col)
+
+    fun up() = Coordinate(row - MOVE_UNIT, col)
+
+    fun down() = Coordinate(row + MOVE_UNIT, col)
+
+    fun left() = Coordinate(row, col - MOVE_UNIT)
+
+    fun right() = Coordinate(row, col + MOVE_UNIT)
+
+    fun isOnBoard(height: Int, width: Int): Boolean {
+        return row in ROW_START_POSITION until height && col in COL_START_POSITION until width
+    }
+
     companion object {
-        const val ROW_START_POSITION = 1
-        const val COL_START_POSITION = 1
+        const val ROW_START_POSITION = 0
+        const val COL_START_POSITION = 0
+        const val MOVE_UNIT = 1
```

Comment:
```suggestion
        private const val MOVE_UNIT = 1
```
클래스 내부에서만 접근한다면 private 접근자를 붙여보면 어떨까요?

Reply:
네 반영했습니다 👍😀